eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -23,7 +23,9 @@ steps:
         succeeded(),
         not(eq(variables['${{ parameters.DependenciesVariableName }}'], ''))
       )
-
+  
+  # $(VCPKG_DEFAULT_TRIPLET) added to the restore key of the cache to make it more unique and force a cache miss
+  # TODO: Use  RFC: Binarycaching in vcpkg can be used for more flexible and less error-prone approach
```

Comment:
Please remove the `#TODO` comment here. I've filed this issue to track investigating this: https://github.com/Azure/azure-sdk-for-cpp/issues/1765

Reply:
```suggestion
```